packages/amplication-client/src/Application/git/dialogs/GitDialogsContainer.tsx

```diff
@@ -51,7 +53,8 @@ export default function GitDialogsContainer({
         onDismiss={handleSelectRepoDialogDismiss}
       >
         <GitRepos
-          applicationId={app.id}
+          applicationId = {app.id}
+          gitOrganizationId={gitOrganizationId}
           onCompleted={handleSelectRepoDialogDismiss}
           sourceControlService={sourceControlService}
```

Comment:
I think this should be called "gitProvider" but if it is too complicated to make the change now we may leave it as it is